person.rb

```diff
@@ -0,0 +1,19 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = rand 1..1000
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+  attr_reader :id
+  attr_accessor :name, :age
```

Comment:
- [ ] Move these 2 object attributes above the `initialize` method. This is considered a standard in OOP in ruby

Reply:
Thanks for sharing this, I just learned this concept.